docs/source/basicusage.rst

```diff
@@ -67,45 +71,55 @@ Example notifications plus help with all available arguments:
   [I 180315 09:27:09 _observer:131] Starting factory <twisted.web.server.Site object at 0x110619828>
   [I 180315 09:27:09 np-api-server:11] Starting REST api server on http://0.0.0.0:8088
 
-  # view help
-  $ np-api-server -h
-  usage: np-api-server [-h]
+View help with all available arguments:
+
+::
+
+    $ np-api-server -h
+    usage: np-api-server [-h]
                      (--mainnet | --testnet | --privnet | --coznet | --config CONFIG)
                      [--port-rpc PORT_RPC] [--port-rest PORT_REST]
                      [--logfile LOGFILE] [--syslog] [--syslog-local [0-7]]
                      [--disable-stderr] [--datadir DATADIR]
                      [--maxpeers MAXPEERS] [--wallet WALLET] [--host HOST]
+                     [--extended-rpc]
 
     optional arguments:
-    -h, --help            show this help message and exit
-    --datadir DATADIR     Absolute path to use for database directories
-    --maxpeers MAXPEERS   Max peers to use for P2P Joining
-    --wallet WALLET       Open wallet. Will allow you to use methods that
-                          require an open wallet
-    --host HOST           Hostname ( for example 127.0.0.1)
+      -h, --help            show this help message and exit
+      --datadir DATADIR     Absolute path to use for database directories
+      --maxpeers MAXPEERS   Max peers to use for P2P Joining
+      --wallet WALLET       Open wallet. Will allow you to use methods that
+                            require an open wallet
+      --host HOST           Hostname ( for example 127.0.0.1)
+      --extended-rpc        Use extended json-rpc api
```

Comment:
This is not the most optimal place to comment, but it will also affect this document. 
> Use extended json-rpc api

this description is not self-explanatory, what does "extended" mean for somebody that just typed `np-prompt -h`. It would be great if we could update this to something like:
> Enable unofficial RPC API extensions

Reply:
https://github.com/jseagrave21/neo-python/blob/96e70141d6b5c4d3961b120a499bb2686f96fa85/neo/bin/api_server.py#L132